src/02-data-types.js

```diff
@@ -127,8 +158,12 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
-
+function isTruthy(val) {
+  if(val){
```

Comment:
Remember we can use `!!` operator to check the inherent truthiness of a value. So here we can just return `!!val`.